solana/src/test/java/org/p2p/solanaj/programs/MemoProgramTest.kt

```diff
@@ -0,0 +1,20 @@
+package org.p2p.solanaj.programs
+
+import org.junit.Assert
+import org.junit.Test
+import kotlin.test.assertEquals
+import org.p2p.solanaj.core.PublicKey
+
+internal class MemoProgramTest {
+
+    private val publicKey = PublicKey("11111111111111111111111111111111")
+
+    @Test
+    fun `memo instruction`() {
```

Comment:
GIVEN memo and pubkey WHEN create memo instruction THEN memo instruction is created